### PR TITLE
refactor: clean up docker cmds and improve code blocks

### PR DESCRIPTION
**Pull request by vinceAmstoutz**
Partially solves #1995 by solving these 2 subtasks : 
- "Remove all `docker compose` prefixes for commands, as we don't use Docker for Laravel"
- "Explain how to run commands with Symfony and Docker in the `symfony/index.md`"

:warning:  Don't forget to check the checkbox in the issue if the PR is merged

### CODE REVIEW

**Review by dunglas — CHANGES_REQUESTED** [review]

File: admin/getting-started.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ Otherwise, follow this guide.
 
 If you don't have an existing React Application, create one using [Create React App](https://create-react-app.dev/):
 
-```console
+```bash
```
[the remark this answers is not in the document]
Comment (vinceAmstoutz) [reply admin/getting-started.md]: Done

File: core/jwt.md [hunk 2]
```diff
@@ -14,11 +17,17 @@ API Platform allows to easily add a JWT-based authentication to your API using [
...
 
 ```console
-docker compose exec php \
-    composer require lexik/jwt-authentication-bundle
+composer require lexik/jwt-authentication-bundle
 ```
+Then we need to generate the public and private keys used for signing JWT tokens.
 
-Then we need to generate the public and private keys used for signing JWT tokens. If you're using the [API Platform distribution](../symfony/index.md), you may run this from the project's root directory:
+You can use LexikJWTAuthenticationBundle to generate them using:
```
Comment (alanpoulain) [inline core/jwt.md:24]:
```suggestion
You can generate them by using this command:
```

File: core/jwt.md [hunk 3]
```diff
@@ -30,13 +39,15 @@ docker compose exec php sh -c '
...
-
-This takes care of keypair creation (including using the correct passphrase to encrypt the private key), and setting the correct permissions on the keys allowing the web server to read them.
+Note that the `setfacl` command relies on the `acl` package. This is installed by default when using the API Platform
+docker distribution but may need to be installed in your working environment in order to execute the `setfacl` command.
 
-Since these keys are created by the `root` user from a container, your host user will not be able to read them during the `docker compose build caddy` process. Add the `config/jwt/` folder to the `api/.dockerignore` file so that they are skipped from the result image.
+This takes care of keypair creation (including using the correct passphrase to encrypt the private key), and setting the
+correct permissions on the keys allowing the web server to read them.
 
-If you want the keys to be auto generated in `dev` environment, see an example in the [docker-entrypoint script of api-platform/demo](https://github.com/api-platform/demo/blob/master/api/docker/php/docker-entrypoint.sh).
```
Comment (alanpoulain) [inline core/jwt.md:39]: Maybe we can keep it and add a permalink instead?

File: schema-generator/getting-started.md [hunk 4]
```diff
@@ -2,15 +2,15 @@
 
 ## Installation
 
-If you use [the API Platform Symfony variant](../symfony/index.md), the Schema Generator is already installed as a development
-dependency of your project and can be invoked through Docker:
+If you use [the API Platform Distribution with Symfony](../symfony/index.md), the Schema Generator is already installed as a development
+dependency of your project and can be invoked with :
```
Comment (alanpoulain) [inline schema-generator/getting-started.md:6]:
```suggestion
dependency of your project and can be invoked with:
```

File: core/jwt.md (hunk 3 above)
Comment (vinceAmstoutz) [reply core/jwt.md:39]: @alanpoulain I'd like to, but I couldn't find the file. I tried again without success. Can you provide me with the link?

Comment (alanpoulain) [reply core/jwt.md:39]: Here: https://github.com/api-platform/demo/blob/a03ce4fb1f0e072c126e8104e42a938bb840bffc/api/docker/php/docker-entrypoint.sh#L16-L17

Comment (vinceAmstoutz) [reply core/jwt.md:39]: Done! @alanpoulain

**Review by alanpoulain — APPROVED** [review]

**Pull request merged**